server/test/controllers/dev/seeding/DevDatabaseSeedControllerTest.java

```diff
@@ -84,7 +89,7 @@ public void seedAndClearCache_clearsCache() {
     assertThat(result.flash().get("success")).hasValue("The database has been seeded");
     result = controller.index(addCSRFToken(fakeRequest()).build());
     assertThat(result.status()).isEqualTo(OK);
-    assertThat(contentAsString(result)).contains("comprehensive-sample-program");
+    assertThat(contentAsString(result)).doesNotContain("comprehensive-sample-program");
```

Comment:
**suggestion (blocking):** instead of just inverting this assertion and the one on L54, add a new test case that ensures the new page you added works

Reply:
My mistake, you did test the new functionality